resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/SimpleResourceService.java

```diff
@@ -77,6 +85,13 @@ private LeafResource asFileResource(final File file) {
                 .connectionDetail(new SimpleConnectionDetail().serviceName(this.dataServiceName));
     }
 
+    /**
+     * Query stream.
+     *
+     * @param uri  the uri
+     * @param pred the pred
+     * @return the stream
```

Comment:
can you be a bit more descriptive on these

Reply:
updated description